homeassistant/components/binary_sensor/egardia.py

```diff
@@ -0,0 +1,98 @@
+"""
+Interfaces with Egardia/Woonveilig alarm control panel.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.egardia/
+"""
+import logging
+import asyncio
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.const import (STATE_ON, STATE_OFF)
+_LOGGER = logging.getLogger(__name__)
+ATTR_DISCOVER_DEVICES = 'egardia_sensor'
+D_EGARDIASYS = 'egardiadevice'
+
+EGARDIA_TYPE_TO_DEVICE_CLASS = {'IR Sensor': 'motion',
+                                'Door Contact': 'opening',
+                                'IR': 'motion'}
+
+
+def _get_device_class(egardia_type):
+    return EGARDIA_TYPE_TO_DEVICE_CLASS.get(egardia_type, None)
+
+
+def _get_sensor_state(egardia_input):
```

Comment:
Just inline this in a variable:
```py
state = STATE_ON if egardia_input else STATE_OFF
```
Remove the function.

Reply:
Done.